vast/nonlinear.go

```diff
@@ -32,10 +32,12 @@ func (nonLinear *NonLinear) Validate() error {
 			}
 		}
 	}
-	if err := nonLinear.MinSuggestedDuration.Validate(); err != nil && err != ErrDurationEqualZero {
-		ve, ok := err.(ValidationError)
-		if ok {
-			errors = append(errors, ve.Errs...)
+	if nonLinear.MinSuggestedDuration != nil {
```

Comment:
Isn't it possible to not have MinSuggestedDuration as a pointer type?

type NonLinear struct {
.....
.....
MinSuggestedDuration      *Duration `xml:"minSuggestedDuration,attr,omitempty"`
.....
.....
} 

I believe the xml parsing should not get affected by the pointer type.

Reply:
I believe we use pointer because it is not a required value, so when we unmarshal the data to the object it will be null instead of 0.